docs/docs/contribute/software/dummy-code-example.go

```diff
@@ -0,0 +1,93 @@
+ // Inside the dummy package
+
```

Comment:
it makes sense to leave this code here to show that it works, but we need a kuttl test to prove that the provider is picked up, that can be used as an example test in the docs and finally we can get rid of the code from the operator and let it live only in the docs

Reply:
Alright.